packages/sdk-js/src/feature-repository.ts

```diff
@@ -289,8 +301,13 @@ function startAutoRefresh(instance: GrowthBook): void {
       src: null,
       cb: (event: MessageEvent<string>) => {
         try {
-          const json: FeatureApiResponse = JSON.parse(event.data);
-          onNewFeatureData(key, json);
+          if (serverSideEval) {
+            const json: { update: boolean } = JSON.parse(event.data);
+            if (json.update) fetchFeatures(instance);
```

Comment:
What happens here? Do we need to call `onNewFeatureData` or similar for changes to take affect?

Reply:
no, `onNewFeatureData` lets us take in a full sdk payload over the SSE message and ingest it. In our case (serverSideEval===true), no payload is coming through, Instead `{ update: true }` comes through, which informs that SDK that a new payload is available. The SDK will then manually fetch that payload (server-side-evaluated).